Box.V2/Config/Constants.cs

```diff
@@ -71,7 +71,8 @@ public static class Constants
         public const string AllFileMetadataPathString = @"{0}/metadata";
         public const string MetadataTemplatesPathString = @"{0}/{1}/schema";
         public const string EnterpriseMetadataTemplatesPathString = @"{0}";
-        public const string TasksPathString = @"{0}/tasks";
+		public const string TasksPathString = @"{0}/tasks";
```

Comment:
spacing, looks like you have tab mixed with space?
Let's use 4 spaces and no tab in our code